pkgs/by-name/ge/gensort/package.nix

```
@@ -0,0 +1,57 @@
+{
+  fetchurl,
+  lib,
+  pkgs,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  name = "gensort";
+  version = "1.5";
+
+  src = fetchurl {
+    url = "https://www.ordinal.com/try.cgi/gensort-${version}.tar.gz";
+    sha256 = "sha256-f3VzeD2CmM7z3Uqh24IlyRTeGgz+0oOWXqILaYOKZ60=";
+  };
+
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+    tar xvf $src
+    runHook postUnpack
+  '';
+
+  nativeBuildInputs = [
+    pkgs.gcc
+    pkgs.zlib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    cd ${name}-${version}
+    make CFLAGS="-O3 -Wall -Wno-unused-result -Wno-maybe-uninitialized -Wno-format-security"
+    install -Dm755 gensort $out/bin/gensort
+    install -Dm755 valsort $out/bin/valsort
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Generate and validate records for the sorting benchmark";
+    longDescription = ''
+      The gensort program can be used to generate input records for the sort
+      benchmarks presented on www.sortbenchmark.org.
+
+      The valsort program can be used to validate the sort output file is
+      correct.
+    '';
+    homepage = "https://www.ordinal.com/gensort.html";
+    license = licenses.gpl2;
+    maintainers = [ maintainers.zimeg ];
```

```suggestion
    maintainers = with lib.maintainers; [ zimeg ];
```
Makes adding maintainers easier